tests/test-cases/Group9-VIC-Admin/9-1-VICAdmin-ShowHTML.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation  Test 9-1 - VICAdmin DisplayHTML
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  ${true}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+Default Tags  regression
```

Comment:
We might want to actually consider moving these to Util.robot and pulling this into Group 7 regression test, so that we continue to only create 1 VCH during regression and maximize speed.